components/kyma-environment-broker/README.md

```diff
@@ -0,0 +1,10 @@
+# Kyma Environment Service Broker
+
+## Overview
+
+The Kyma Environment Service Broker provide a way to run Kyma as a service on different providers like: AWS, GCP or Azure.
+
+The broker provides a services which installs Kyma with a separate plan for each provider.
+
+It uses the Compass Provisioner API to install Kyma on the given cluster.
+
```

Comment:
Maybe you could write something more about how to run the broker itself? Some words about the parameters.